### ISSUE TITLE
runstreams panel disappears and can't be recovered

### ISSUE THREAD

**Issue by gjw** [ryrobes/rvbbit#7]
to repro:

- open 'frost-terminal2'
- observe 'runstreams' button at bottom middle between 'flow' and 'editor' buttons
- click 'runstreams' button to restore panel
- observe panel is restored, runstreams and snapshots selectable in top right of panel bar
- select any other block
- observe runstreams panel has switched to 'history' panel, neither runstreams nor snapshots visiable as options
- minimizing etc seems to have no effect, this panel remains the 'history' only panel until refreshing the browser

There well may be some other way to get the runstreams panel back aside from browser refresh, but I can't find it.

**Comment by ryrobes** [comment]
Yeah it's awkward at the moment - but runstreams and snapshots only show when there is not block selected. So if you hit ESC a few times, everything would be unselected and runstreams would return. 

Shift-space also toggles that panel if you didnt want to hit the minimized button.

This is good feedback though, because I had an idea that it was a bit jarring and odd. Perhaps runstreams and snapshots should ALWAYS be avail, but history and other panel-specific options should show up if a block is selected - rather than the current way of having "2 modes" essentially based on if a block is selected or not.

**Comment by gjw** [comment]
Ahh ok.
Yeah least jarring imo would be to always show all, but have content of history be 'No block selected' when there's... no block selected.
